website/templates/nav.mako

```diff
@@ -48,9 +48,9 @@
 
             % if user_name and display_name:
             <li class="dropdown">
-            <button class="dropdown-toggle nav-user-dropdown btn-link" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="Toggle auth dropdown">
+            <button class="dropdown-toggle nav-user-dropdown btn-link" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="Toggle auth dropdown" style="margin-top: 12px;">
                 <span class="osf-gravatar">
-                    <img src="${user_gravatar}" alt="User gravatar">
+                    <img src="${user_gravatar}" style="top: -2px;position: relative;" alt="User gravatar">
```

Comment:
Now chrome is out of line.
 
<img width="388" alt="screen shot 2017-07-12 at 11 13 10 am" src="https://user-images.githubusercontent.com/1322421/28125951-2abfe0bc-66f6-11e7-8d54-ee1e9657e67f.png">